source/operations/monitoring/metrics-and-alerts.rst

```diff
@@ -15,80 +15,87 @@ Metrics and Alerts
 MinIO publishes cluster and node metrics using the :prometheus-docs:`Prometheus Data Model <concepts/data_model/>`.
 You can use any scraping tool to pull metrics data from MinIO for further analysis and alerting.
 
-MinIO provides scraping endpoints for the following metric groups:
+Starting with MinIO Server :minio-release:`RELEASE.2024-07-15T19-02-30Z` and MinIO Client :mc-release:`RELEASE.2024-07-11T18-01-28Z`, metrics version 3 replaces the deprecated :ref:`metrics version 2 <minio-metrics-v2>`.
```

Comment:
BUT ALSO need to confirm the MinIO release. It's after the purportedly corresponding `mc` one?

Reply:
Ok it's because there was a bug fix for the actual metrics implementation _after_ the relevant `mc` release. Keep it this way? 

Or go back to the original "done" release so we don't confuse people about the standard guidance on MinIO vs `mc` release sequence.

Reply:
I'd keep it like it is. We don't want them trying to use a release with a known bug.